### ISSUE TITLE
Error with ldap_escape

### ISSUE THREAD

**Issue by tsmr** [pluginsGLPI/moreldap#13]
Trunk version 

With my Active directory, this call in hook.php : 

```
$userdn          = ldap_escape(isset($user->input["user_dn"])
                                       ? $user->input["user_dn"]
                                       : $user->fields["user_dn"]);
```

give me php-error & stop import : 

```
2016-05-31 10:29:51 [6@P365]
  *** PHP Warning(2): ldap_read(): Search: Invalid DN syntax
  Backtrace :
  :                                                  
  plugins\moreldap\hook.php:106                      ldap_read()
  :                                                  plugin_moreldap_item_add_or_update_user()
  inc\plugin.class.php:1095                          call_user_func()
  inc\commondbtm.class.php:1123                      Plugin::doHook()
  inc\authldap.class.php:2101                        CommonDBTM->update()
  front\user.form.php:107                            AuthLDAP::ldapImportUserByServerId()
2016-05-31 10:29:51 [6@P365]
  *** PHP Warning(2): ldap_get_entries() expects parameter 2 to be resource, boolean given
  Backtrace :
  :                                                  
  inc\authldap.class.php:3128                        ldap_get_entries()
  plugins\moreldap\hook.php:107                      AuthLDAP::get_entries_clean()
  :                                                  plugin_moreldap_item_add_or_update_user()
  inc\plugin.class.php:1095                          call_user_func()
  inc\commondbtm.class.php:1123                      Plugin::doHook()
  inc\authldap.class.php:2101                        CommonDBTM->update()
  front\user.form.php:107                            AuthLDAP::ldapImportUserByServerId()
```

With drop ldap_escape : 

```
$userdn          = isset($user->input["user_dn"])
                                       ? $user->input["user_dn"]
                                       : $user->fields["user_dn"];
```

No errors & location is imported

**Comment by btry** [comment]
Hi

https://github.com/btry/moreldap/tree/master%2Bldap_escape

This is WIP. There is an error handling which needs tests. It should log the DNs causing the issue before and after ldap_escape().

**Comment by btry** [comment]
Fixed with #14
